### PR TITLE
Modtool: Add copyright feature

### PR DESCRIPTION
**Pull request by swap-nil7**
The feature for adding `copyrightholder` should be there in the `3.8 release`.

### CODE REVIEW

File: gr-utils/python/modtool/cli/add.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 from __future__ import unicode_literals
 
 import re
+import getpass
```
Comment (mbr0wn) [inline gr-utils/python/modtool/cli/add.py]: Another silent dependency?

File: gr-utils/python/modtool/cli/add.py [hunk 2]
```diff
@@ -109,7 +110,16 @@ def get_blockname(self):
 def get_copyrightholder(self):
     """ Get the copyrightholder of the block to be added """
     if self.info['copyrightholder'] is None:
-        self.info['copyrightholder'] = '<+YOU OR YOUR COMPANY+>'
+        user = getpass.getuser()
```
Comment (mbr0wn) [inline gr-utils/python/modtool/cli/add.py]: I don't think this is worth the cost of another dependency.

Comment (swap-nil7) [reply gr-utils/python/modtool/cli/add.py]: I am not very sure but I think `getpass` is a internal python library.

Comment (mbr0wn) [reply gr-utils/python/modtool/cli/add.py]: Ah alrighty.

File: gr-utils/python/modtool/cli/add.py (hunk 1 above)
Comment (mbr0wn) [reply gr-utils/python/modtool/cli/add.py]: Turns out no.

**Pull request merged**